specification.markdown

```diff
@@ -228,6 +210,95 @@ VINT\_WIDTH | VINT\_MARKER | VINT\_DATA     | Element Data Size Status
         For example -2 can be coded as 0xFFFFFFFFFFFFFE or 0xFFFE or
         0xFE and 5 can be coded 0x000000000005 or 0x0005 or 0x05.
 
+## EBML Element Types
+
+Each defined EBML Element MUST have a declared Element Type. The Element Type defines a concept for storing data that may be constrained by length, endianness, and purpose.
+
+Element Data Type:   Signed Integer
+
+    Endianness:     Big-endian
+    Length:         A Signed Integer Element MUST at least 1 octet in
```

Comment:
This particular corner case issue is prominent because of inconsistencies between Martin Nilsson's RFC Draft of Matroska and the official specs. The draft allowed many zero-signed EBML data types that the official EBML/MKV specs didn't. From what I'm reading there still seems to be consensus for allowing zero-sized elements with integers and floats.